charts/sonarqube-dce/README.md

```diff
@@ -257,7 +257,11 @@ The following table lists the configurable parameters of the SonarQube chart and
 | `searchNodes.persistence.size`                            | Size of the PVC                                                                       | `5G`                                                                   |
 | `searchNodes.persistence.uid`                             | UID used for init-fs container                                                        | `1000`                                                                 |
 | `searchNodes.persistence.guid`                            | GUID used for init-fs container                                                       | `0`                                                                    |
-| `searchNodes.extraContainers`                             | Array of extra containers to run alongside                                            | `[]`                                                                   |
+| `searchNodes.extraContainers`                             | Array of extra containers to run alongside                                            | `[]`                                                                      |
+| `searchNodes.nodeSelector`                                | Node labels for pod assignment for searchNodes, global nodeSelector take precedance                                                                                                                                          | `{}` |
```

Comment:
- "Node labels for pod assignment for searchNodes" -> "Node labels for search nodes' pods assignment"
- take -> takes (please propagate to the rest)